common/app/components/Nav/index.js

```diff
@@ -1 +1 @@
-export default from './Nav.jsx';
```

Comment:
Ah I didn't know about that proposal! I like the simplification. I'll remove these changes and update the other two to be * as v syntax. I don't know the proposal structure how long does a stage 1 proposal take to get through?